src/components/UI/atoms/Image/Image.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { borderRadius } from '<variables>/border';
+import { boxShadows } from '<variables>/shadows';
+import { borderColors } from '<variables>/colorPalette';
+
+/**
+ * @description - Image Component
+ *
+ * @prop {string} width - image width
+ * @prop {string} height - image height
+ * @prop {string} ImageUrl - image source
+ * @prop {string} altText - alt text
+ * @prop {string} boxShadow - shadowType
+ * @prop {boolean} borderColor- borderColor
+ * @prop {string} borderRadiusType - borderRadiusType
+ * @return {component} Image
+ */
+
+const Image = (
+  {
+    ImageUrl, altText, width, height, boxShadow, borderRadiusType, borderColor,
+  },
+) => {
+  return <Image.Container
+    src={ImageUrl}
+    alt={altText}
+    width={width}
+    height={height}
+    boxShadow={boxShadow}
+    borderRadiusType={borderRadiusType}
+    borderColor={borderColor}/>;
+};
+Image.Container = styled.img`
+${({
+    width,
+    height,
+    boxShadow,
+    borderRadiusType,
+    borderColor,
+    theme,
+  }) => {
+    return `
+  width: ${width} || '100%';
+  min-height: ${height} || '100%';
+  box-shadow:${theme.boxShadows[boxShadow]};
+  border: ${theme.borderColors[borderColor]};
+  border-radius:${theme.borderRadius[borderRadiusType]};
+  `;
+  }}`;
+
+Image.propTypes = {
+  ImageUrl: PropTypes.string.isRequired,
+  altText: PropTypes.string,
```

Comment:
It is good practice to do an altText this is particularly good for accessibility for blind users